pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -431,21 +431,22 @@ func updateIsEquivalent(a configv1.Update, b configv1.Release) bool {
 	}
 }
 
-// sortSemanticVersions sorts the input slice in increasing order.
-func sortSemanticVersions(versions []configv1.Release) {
+// sortReleasesBySemanticVersions sorts the input slice in decreasing order.
```

Comment:
Not sure why would we want to change the sort order - just for the sake of using `cv.Status.AvailableUpdates[0]` instead of `len(cv.Status.AvailableUpdates)-1`?

Reply:
I have dropped the initial changes for modifying the sorting functions.
(The initial thinking was to have a decreasing order of the semantic versions. And then have the following invalid semantic versions in ascending lexicographic order. Which would not be possible with reversing the current sorting function returned slice.)
And I have added functions for reversing the relevant slices to be in decreasing order.